include/seqan/seq_io/fasta_fastq.h

```diff
@@ -413,7 +416,11 @@ inline void readRecord(TIdString & meta, TSeqString & seq, TQualString & qual, T
     TQualIgnoreOrAssert qualIgnore;
 
     readUntil(qual, iter, qualCountDown, qualIgnore);  // read Fastq qualities
-    skipUntil(iter, TFastqBegin());     // forward to the next '@'
+    skipUntil(iter, qualCountDown);                     // skip Fastq qualities
+    skipUntil(iter, NotFunctor<IsWhitespace>());        // skip Fastq qualities
```

Comment:
I don't understand the first `skipUntil()`. Why are you skipping if you are reading before? Please also fix the comments!